packages/components/src/search-list-control/index.js

```diff
@@ -138,7 +138,7 @@ export class SearchListControl extends Component {
 					depth,
 				} ) }
 				{ this.renderList( item.children, depth + 1 ) }
-			</Fragment>
+			</li>
```

Comment:
You should keep `this.renderList( item.children, depth + 1 )` outside the `<li>`, otherwise list items are getting nested one inside the other.

I think something like this should work:

```JSX
			<Fragment key={ item.id }>
				<li>
					{ renderItem( {
						item,
						isSelected: this.isSelected( item ),
						onSelect: this.onSelect,
						isSingle,
						search,
						depth,
					} ) }
				</li>
				{ this.renderList( item.children, depth + 1 ) }
			</Fragment>
```

Reply:
Thank you Albert!! The change has been pushed.